lib/webvalve/railtie.rb

```diff
@@ -1,23 +1,23 @@
 module WebValve
   class Railtie < ::Rails::Railtie
-    if WebValve.enabled?
-      initializer 'webvalve.set_autoload_paths', before: :set_autoload_paths do |app|
-        WebValve.config_paths << app.root
+    initializer 'webvalve.set_autoload_paths', before: :set_autoload_paths do |app|
+      return unless WebValve.enabled?
```

Comment:
this seems fine. i was trying to avoid even registering the initializers if the library was not going to be used, but i guess this is okay as well. 

i don't personally use figaro, so i've not run into the issue that prompted this change.

Reply:
is there any harm in using `return` here? i haven't dug into the rails initialization process in a while, but i'm a little concerned that return could cause other initializers after this one _not to_ run. have you verified that? if it is an issue we can just use an if statement.

Reply:
No there is no harm. `initialize` creates an instance of `Initializer` which is added to the initializers. Early returning does not effect other initializers.

Reply:
cool! thanks for checking.